src/main/java/com/theironyard/invitator/Invite.java

```diff
@@ -0,0 +1,53 @@
+package com.theironyard.invitator;
+
+/**
+ * Created by melmo on 12/26/16.
+ */
+public class Invite {
+    private Integer invite_id;
+    private Integer event_id;
+    private Integer person_id;
+    private Boolean invited;
+
+
+
+// DEFAULTS --------------------------------------------------
```

Comment:
Typically you would want a different way to do comments instead of 1 line comments.